packages/app-store/googlecalendar/api/callback.ts

```diff
@@ -80,6 +80,7 @@ async function getHandler(req: NextApiRequest, res: NextApiResponse) {
     const gCalService = new GoogleCalendarService({
       ...gcalCredential,
       user: null,
+      delegatedTo: null,
```

Comment:
This endpoint is only part of non-dwd flow, so we set `delegatedTo: null` explicitly